pygmt/src/grdtrack.py

```diff
@@ -37,10 +37,10 @@ def grdtrack(points, grid, newcolname=None, outfile=None, **kwargs):
     r"""
     Sample grids at specified (x,y) locations.
 
-    Reads one or more grid files and a table (from file or an array input; but
+    Reads a grid file and a table (from file or an array input; but
     see ``profile`` for exception) with (x,y) [or (lon,lat)] positions in the
     first two columns (more columns may be present). It interpolates the
-    grid(s) at the positions in the table and writes out the table with the
+    grid at the positions in the table and writes out the table with the
     interpolated values added as (one or more) new columns. Alternatively
```

Comment:
Is adding more than one new column possible with a single grid input, which I would assume could happen for a grid input with more than 3 dimensions? Or could this only be the case with multiple grid inputs in a single function?

Reply:
One could set `crossprofile`/[`-C`](https://docs.generic-mapping-tools.org/6.1/grdtrack.html#c) and get more columns (with just a single grid as input), but this would only work for file outputs, as 3 columns are currently assumed for pandas.DataFrame output (lon, lat, z).